src/components/NavItems.js

```diff
@@ -0,0 +1,48 @@
+import { Flex, Text, Icon, Link, Menu, MenuButton, MenuList } from '@chakra-ui/react';
+import NavHoverBox from '../components/NavHoverBox';
```

Comment:
We want to use absolute imports everywhere, that's why we added that `jsconfig.json` file - maybe you need to add the aliases there to be able to use it like this:

```suggestion
import NavHoverBox from '@components/NavHoverBox';
```

Example:
![image](https://user-images.githubusercontent.com/5387195/147361592-e220e56a-5189-4598-8deb-b95ce9096d61.png)

Reply:
I implemented all changes beside this one. This file makes sense to me and I understand it - but if I try using it, it fails to compile with following error: "Field 'browser' doesn't contain a valid alias configuration". I read couple of articles on this topic, and it looks like that there is some problem with Webpack configuration, but I wasn't sure on how to proceed - I could use some help here :)